src/printer.js

```diff
@@ -2206,20 +2221,32 @@ function printJSXElement(path, options, print) {
   let forcedBreak = false;
 
   // Trim trailing whitespace, recording if there was a hardline
+  let numTrailing = 0;
```

Comment:
@jlongster this is actually the number of trailing \n and not trailing whitespace. So this is important to make sure that if there are three empty lines, now there's only one.

Just changing the description should be good :)